### PR TITLE
Add a readme file for releasing the project

### PR DESCRIPTION
**Pull request by hhorak**
@fila43 @mkulik-rh I tried to put things together, does the steps make sense or should I explain something more?

### CODE REVIEW

File: README.release [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+```
+git push
+```
+
+7. Create a tarball
+```
+aclocal
+automake --add-missing
+autoconf
```
Comment (praiskup) [inline README.release]: autoreconf (-vfi) should do this

Comment (praiskup) [reply README.release:37]: autoreconf (-vfi) should do this

**Comment by praiskup** [comment]
otherwise looks nice

**Comment by fila43** [comment]
It makes sense to me

**Comment by mkulik-rh** [comment]
Looks fine, maybe it would be a good idea to automate it in Github Actions ? That way we could automatically build and release new version when we push a new tag.

**Comment by hhorak** [comment]
> Looks fine, maybe it would be a good idea to automate it in Github Actions ? That way we could automatically build and release new version when we push a new tag.

That's interesting idea. On the other hand if the manual release takes us few minutes every several months, I'm not sure whether it's worth the time spending on automating it.

File: README.release (hunk 1 above)
Comment (hhorak) [reply README.release]: Thanks for the suggestion, will use this.

**Pull request merged**